Solutions/Apache Log4j Vulnerability Detection/Hunting Queries/Linux_Toolkit_Detected.yaml

```diff
@@ -1,8 +1,9 @@
 id: 09e45ec6-ac42-4b5a-be69-54623c4aa062
 name: Possible Linux attack toolkit detected via Syslog data
 description: |
-  'This query uses syslog data to alert on any attack toolkits associated with massive scanning or exploitation attempts against a known vulnerability.
-  Attackers may perform such operations as seen recently to exploit the remote code execution vulnerability in Log4j component of Apache to scope and prioritize post-compromise objectives.
+   'This query alerts on attack toolkits associated with massive scanning or exploitation attempts against a known vulnerability. Attackers may exploit remote code execution vulnerability in Log4j component of Apache to prioritize post-compromise objectives'
```

Comment:
'This query searches for usage of attack toolkits associated with massive scanning or exploitation of remote code execution vulnerability in Log4j component of Apache.'

Reply:
Resolved